core/ulog/README.md

```diff
@@ -1,8 +1,8 @@
 # Logging package
 
-ulog provides an API wrapper around the logging library (zap Logger)
-The logger is instantiated as logger with default options and can be configured
-via `Configure()` API and provided YAML configuration.
+`package ulog` provides an API wrapper around the logging library (zap Logger).
+`ulog` uses builder pattern to instantiate the logger. Using `LogBuilder` user can set up
```

Comment:
How about - 
"With `LogBuilder` user can do pre-initialization set up by injecting configuration, custom logger and log level prior to building the usable `ulog.Log` object."